From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Wed, 15 Feb 2023 14:38:49 +0300
Subject: General Makefile

---
 Makefile | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..ec7d7c2
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,17 @@
+# General Makefile for Debian packaging system.
+
+DESTDIR =
+
+all:
+	$(MAKE) -C src all
+	$(MAKE) -C lib all
+
+install:
+	$(MAKE) -C src install prefix=$(DESTDIR)/usr
+	$(MAKE) -C lib install prefix=$(DESTDIR)/usr
+
+clean:
+	$(MAKE) -C src clean
+	$(MAKE) -C lib clean
+
+.PHONY: clean install
