diff -u text710.edin.orig text710.edin
--- text710.edin.orig	2011-05-29 04:10:25.000000000 +0400
+++ text710.edin	2011-05-29 06:10:39.000000000 +0400
@@ -1009,7 +1009,6 @@
 Dingle d * i ng g l
 Dinnington d * i n i ng t a n
 Dip d * i p
-Dir d i . r * e k t a r
 Dives d * ie v ee z
 Djibouti j i . b * uu t i
 Djiboutian j i . b * uu t eer n
@@ -1434,6 +1433,7 @@
 Gov g * u v a n a r
 Governor-General g ~ u v n a - j * e n r a l
 Governor-Generals g ~ u v n a - j * e n r a l z
+Graham g r * aa m
 Grampian g r * aa m p eer n
 Granada g r a . n * ar d a
 Grangemouth g r * ai n j m a th
@@ -2596,7 +2596,6 @@
 Nicaragua n ~ i k a . r * aa g y oor
 Nicaraguan n ~ i k a . r * aa g y oor n
 Nicaraguans n ~ i k a . r * aa g y oor n z
-Nice n * ee s
 Nicholas n * i k a l a s
 Nick n * i k
 Nicola n * i k a l a
@@ -4200,6 +4199,7 @@
 abstracting a b . s t r * aa k t i ng
 abstraction a b . s t r * aa k sh n
 abstractions a b . s t r * aa k sh n z
+abstractness a b . s t r * aa k t n a s
 abstracts * aa b s t r aa k t s
 abstracts a b . s t r * aa k t s
 abstruse a b . s t r * uu s
@@ -7037,7 +7037,6 @@
 attaboy ~ aa t a . b * oi
 attach a . t * aa ch
 attache a . t * aa sh ai
-attaches a . t * aa sh ai z
 attached a . t * aa ch t
 attaches a . t * aa ch i z
 attaching a . t * aa ch i ng
@@ -8526,10 +8525,13 @@
 behaved b i . h * ai v d
 behaves b i . h * ai v z
 behaving b i . h * ai v i ng
+behavior b i . h * ai v eer r
+behaviors b i . h * ai v eer z
 behaviour b i . h * ai v eer r
 behaviourism b i . h * ai v eer r i z a m
 behaviourist b i . h * ai v eer r i s t
 behaviourists b i . h * ai v eer r i s t s
+behaviours b i . h * ai v eer z
 behead b i . h * e d
 beheaded b i . h * e d i d
 beheading b i . h * e d i ng
@@ -9605,6 +9607,7 @@
 bluest b l * uu i s t
 bluestocking b l * uu s t o k i ng
 bluestockings b l * uu s t o k i ng z
+bluetooth b l * uu t uu th
 bluff b l * u f
 bluffed b l * u f t
 bluffer b l * u f a r
@@ -10690,6 +10693,9 @@
 brows b r * ou z
 browse b r * ou z
 browsed b r * ou z d
+browser b r * ou z a r
+browserless b r * ou z a l a s
+browsers b r * ou z a z
 browses b r * ou z i z
 browsing b r * ou z i ng
 bruin b r * uu i n
@@ -11221,6 +11227,8 @@
 byroads b * ie r oa d z
 bystander b * ie s t aa n d a r
 bystanders b * ie s t aa n d a z
+byte b * ie t
+bytes b * ie t s
 byway b * ie w ai
 byways b * ie w ai z
 byword b * ie w er d
@@ -14297,7 +14305,9 @@
 colonnades k ~ o l a . n * ai d z
 colons k * oa l a n z
 colony k * o l a n i
+color k * u l a r
 coloratura k ~ o l a r a . t * oor r a
+colors k * u l a z
 colossal k a . l * o s l
 colossi k a . l * o s ie
 colossus k a . l * o s a s
@@ -14498,6 +14508,8 @@
 committed k a . m * i t i d
 committee k a . m * i t i
 committees k a . m * i t i z
+committer k a . m * i t a r
+committers k a . m * i t a z
 committing k a . m * i t i ng
 commode k a . m * oa d
 commodes k a . m * oa d z
@@ -14719,6 +14731,7 @@
 composes k a m . p * oa z i z
 composing k a m . p * oa z i ng
 composite k * o m p a z i t
+composites k * o m p a z i t s
 composition k ~ o m p a . z * i sh n
 compositions k ~ o m p a . z * i sh n z
 compositor k a m . p * o z i t a r
@@ -17474,6 +17487,10 @@
 customary k * u s t a m a r i
 customer k * u s t a m a r
 customers k * u s t a m a z
+customizable k u s t a m * ie z a b l
+customize k * u s t a m ie z
+customized k * u s t a m ie z d
+customizes k * u s t a m ie z i z
 customs k * u s t a m z
 cut k * u t
 cut-out k * u t - ou t
@@ -18177,6 +18194,12 @@
 decolonized d ~ ee . k * o l a n ie z d
 decolonizes d ~ ee . k * o l a n ie z i z
 decolonizing d ~ ee . k * o l a n ie z i ng
+decompile d ~ ee k a m . p * ie l
+decompiled d ~ ee k a m . p * ie l d
+decompiler d ~ ee k a m . p * ie l a r
+decompilers d ~ ee k a m . p * ie l a z
+decompiles d ~ ee k a m . p * ie l z
+decompiling d ~ ee k a m . p * ie l i ng
 decompose d ~ ee k a m . p * oa z
 decomposed d ~ ee k a m . p * oa z d
 decomposes d ~ ee k a m . p * oa z i z
@@ -22551,6 +22574,8 @@
 embonpoint ~ o m b o n . p w * ar ng
 emboss i m . b * o s
 embossed i m . b * o s t
+embosser i m . b * o s a r
+embossers i m . b * o s a z
 embosses i m . b * o s i z
 embossing i m . b * o s i ng
 embrace i m . b r * ai s
@@ -22889,6 +22914,7 @@
 enforceable i n . f * aw s a b l
 enforced i n . f * aw s t
 enforcement i n . f * aw s m a n t
+enforcements i n . f * aw s m a n t s
 enforces i n . f * aw s i z
 enforcing i n . f * aw s i ng
 enfranchise i n . f r * aa n ch ie z
@@ -24200,8 +24226,6 @@
 exports * e k s p aw t s
 exports i k . s p * aw t s
 expos * e k s p oa z
-expose e k . s p * oa z ai
-exposes e k . s p * oa z ai z
 expose i k . s p * oa z
 exposed i k . s p * oa z d
 exposes i k . s p * oa z i z
@@ -24486,6 +24510,7 @@
 face-saver f * ai s - s ai v a r
 face-savers f * ai s - s ai v a z
 face-saving f * ai s - s ai v i ng
+facebook f * ai s b oo k
 faced f * ai s t
 faceless f * ai s l a s
 facer f * ai s a r
@@ -25745,6 +25770,8 @@
 flaunts f l * aw n t s
 flautist f l * aw t i s t
 flautists f l * aw t i s t s
+flavor f l * aa v a r
+flavors f l * aa v a z
 flavour f l * ai v a r
 flavoured f l * ai v a d
 flavouring f l * ai v a r i ng
@@ -27265,6 +27292,7 @@
 fun f * u n
 function f * u ng k sh n
 functional f * u ng k sh a n l
+functionalities f ~ u ng k sh a n * aa l i t i z
 functionalism f * u ng k sh a n a l i z a m
 functionalist f * u ng k sh n a l i s t
 functionalists f * u ng k sh n a l i s t s
@@ -28111,6 +28139,8 @@
 gifting g * i f t i ng
 gifts g * i f t s
 gig g * i g
+gigabyte g * i g a b ie t
+gigabytes g * i g a b ie t s
 gigantic j ie . g * aa n t i k
 giggle g * i g l
 giggled g * i g l d
@@ -30695,6 +30725,7 @@
 hieroglyphics h ~ ie a r a . g l * i f i k s
 hieroglyphs h * ie a r a g l i f s
 hies h * ie z
+hifi h ie f * ie
 higgledy-piggledy h ~ i g l d i - p * i g l d i
 high h * ie
 high-class h * ie - k l ar s
@@ -35214,6 +35245,8 @@
 kilns k * i l n z
 kilo k * ee l oa
 kilo- k * i l a -
+kilobyte k * i l a b ie t
+kilobytes k * i l a b ie t s
 kilocycle k * i l a s ie k l
 kilocycles k * i l a s ie k l z
 kilogram k * i l a g r aa m
@@ -36532,6 +36565,8 @@
 lifeboats l * ie f b oa t s
 lifebuoy l * ie f b oi
 lifebuoys l * ie f b oi z
+lifecycle l * ie f s ie k l
+lifecycles l * ie f s ie k l z
 lifeguard l * ie f g ar d
 lifeguards l * ie f g ar d z
 lifeless l * ie f l a s
@@ -36542,6 +36577,8 @@
 lifelong l * ie f l o ng
 lifer l * ie f a r
 lifers l * ie f a z
+lifestyle l * ie f s t ie l
+lifestyles l * ie f s t ie l z
 lifetime l * ie f t ie m
 lifetimes l * ie f t ie m z
 lift l * i f t
@@ -37857,6 +37894,7 @@
 make-ups m * ai k - u p s
 maker m * ai k a r
 makers m * ai k a z
+makefile m ~ ai k . f * ie l
 makes m * ai k s
 makeshift m * ai k sh i f t
 makeshifts m * ai k sh i f t s
@@ -38780,6 +38818,8 @@
 meeting-places m * ee t i ng - p l ai s i z
 meetings m * ee t i ng z
 meets m * ee t s
+megabyte m * e g a b ie t
+megabytes m * e g a b ie t s
 megacycle m * e g a s ie k l
 megacycles m * e g a s ie k l z
 megadeath m * e g a d e th
@@ -39010,6 +39050,7 @@
 mess-ups m * e s - u p s
 message m * e s i j
 messages m * e s i j i z
+messaging m * e s i j i ng
 messed m * e s t
 messenger m * e s i n j a r
 messengers m * e s i n j a z
@@ -40676,6 +40717,7 @@
 multiplies m * u l t i p l ie z
 multiply m * u l t i p l ie
 multiplying m * u l t i p l ie i ng
+multitouch m ~ u l t i . t * u ch
 multitude m * u l t i t y uu d
 multitudes m * u l t i t y uu d z
 multitudinous m ~ uu l t i . t y * uu d i n a s
@@ -41065,6 +41107,7 @@
 nations n * ai sh n z
 nationwide n * ai sh n w ie d
 native n * ai t i v
+natively n * ai t i v l i
 natives n * ai t i v z
 nativities n a . t * i v i t i z
 nativity n a . t * i v i t i
@@ -41689,6 +41732,7 @@
 nonentities n o . n * e n t i t i z
 nonentity n o . n * e n t i t i
 nonesuch n * u n s u ch
+nonetheless n ~ u n dh a . l * e s
 nonevent n * o n i v e n t
 nonevents n * o n i v e n t s
 nonfiction n o n . f * i k sh n
@@ -42388,6 +42432,7 @@
 officiousness a . f * i sh a s n a s
 offing * o f i ng
 offish * o f i sh
+offline o f . l * ie n
 offprint * o f p r i n t
 offprints * o f p r i n t s
 offset * o f s e t
@@ -42538,6 +42583,7 @@
 ongoing * o n g oa i ng
 onion * u n eer n
 onions * u n eer n z
+online o n . l * ie n
 onlooker * o n l oo k a r
 onlookers * o n l oo k a z
 only * oa n l i
@@ -42654,6 +42700,7 @@
 opposes a . p * oa z i z
 opposing a . p * oa z i ng
 opposite * o p a z i t
+oppositely * o p a z i t l i
 opposites * o p a z i t s
 opposition ~ o p a . z * i sh n
 oppress a . p r * e s
@@ -47087,6 +47134,7 @@
 port p * aw t
 portability p ~ aw t a . b * i l i t i
 portable p * aw t a b l
+portably p * aw t a b l i
 portage p * aw t i j
 portages p * aw t i j i z
 portal p * aw t l
@@ -49773,8 +49821,6 @@
 rechauffes r ai . sh * oa f ai z
 regime r ai . zh * ee m
 regimes r ai . zh * ee m z
-resume r * e z y uu m ai
-resumes r * e z y uu m ai z
 rabbi r * aa b ie
 rabbinical r a . b * i n i k l
 rabbis r * aa b ie z
@@ -50177,6 +50223,12 @@
 rasping r * ar s p i ng
 raspingly r * ar s p i ng l i
 rasps r * ar s p s
+rasterize r * aa s t a r ie z
+rasterized r * aa s t a r ie z d
+rasterizer r * aa s t a r ie z a r
+rasterizers r * aa s t a r ie z a z
+rasterizes r * aa s t a r ie z i z
+rasterizing r * aa s t a r ie z i ng
 rat r * aa t
 rat-a-tat-tat r ~ aa t - a - t ~ aa t - t * aa t
 rat-a-tat-tats r ~ aa t - a - t ~ aa t - t * aa t s
@@ -50394,6 +50446,7 @@
 readjustment r ~ ee a . j * u s t m a n t
 readjustments r ~ ee a . j * u s t m a n t s
 readjusts r ~ ee a . j * u s t s
+readme r * ee d . m ee
 readmission r ~ ee a d . m * i sh n
 readmissions r ~ ee a d . m * i sh n z
 readmit r ~ ee a d . m * i t
@@ -50556,7 +50609,6 @@
 rebuttals r i . b * u t a l z
 rebutted r i . b * u t i d
 rebutting r i . b * u t i ng
-rec r i . s * ee v d
 recalcitrance r i . k * aa l s i t r a n s
 recalcitrancy r i . k * aa l s i t r a n s i
 recalcitrant r i . k * aa l s i t r a n t
@@ -50636,6 +50688,11 @@
 recessionals r i . s * e sh a n l z
 recessions r i . s * e sh n z
 recessive r i . s * e s i v
+recharge r i . ch * ar j
+rechargeable r i . ch * ar j a b l
+recharged r i . ch * ar j d
+recharges r i . ch * ar j i z
+recharging r i . ch * ar j i ng
 recherche r a . sh * air sh ai
 recidivism r i . s * i d i v i z a m
 recidivist r i . s * i d i v i s t
@@ -50719,6 +50776,10 @@
 recompensed r * e k a m p e n s t
 recompenses r * e k a m p e n s i z
 recompensing r * e k a m p e n s i ng
+recompile r ~ ee k a m . p * ie l
+recompiled r ~ ee k a m . p * ie l d
+recompiles r ~ ee k a m . p * ie l z
+recompiling r ~ ee k a m . p * ie l i ng
 reconcilable r ~ e k a n . s * ie l a b l
 reconcile r * e k a n s ie l
 reconciled r * e k a n s ie l d
@@ -51995,6 +52056,8 @@
 reseeding r ee . s * ee d i ng
 reseeds r ee . s * ee d z
 resell r ee . s * e l
+reseller r ee . s * e l a r
+resellers r ee . s * e l a z
 reselling r ee . s * e l i ng
 resells r ee . s * e l z
 resemblance r i . z * e m b l a n s
@@ -53571,6 +53634,7 @@
 runny r * u n i
 runs r * u n z
 runt r * u n t
+runtime r ~ u n . t * ie m
 runts r * u n t s
 runway r * u n w ai
 runways r * u n w ai z
@@ -54658,6 +54722,9 @@
 screen s k r * ee n
 screened s k r * ee n d
 screening s k r * ee n i ng
+screenreader s k r * ee n r ee d a r
+screenreaders s k r * ee n r ee d a z
+screenreading s k r * ee n r ee d i ng
 screens s k r * ee n z
 screes s k r * ee z
 screw s k r * uu
@@ -54902,6 +54969,7 @@
 seamiest s * ee m i i s t
 seaming s * ee m i ng
 seamless s * ee m l a s
+seamlessly s * ee m l a s l i
 seams s * ee m z
 seamstress s * ee m s t r i s
 seamstresses s * ee m s t r i s i z
@@ -57055,6 +57123,7 @@
 skylights s k * ie l ie t s
 skyline s k * ie l ie n
 skylines s k * ie l ie n z
+skype s k * ie p
 skyrocket s k * ie r o k i t
 skyrocketed s k * ie r o k i t i d
 skyrocketing s k * ie r o k i t i ng
@@ -60716,6 +60785,7 @@
 subpoenaed s a . p * ee n a d
 subpoenaing s a . p * ee n a r i ng
 subpoenas s a . p * ee n a z
+subroutine s ~ u b . r uu . t * ee n
 subs s * u b z
 subscribe s a b . s k r * ie b
 subscribed s a b . s k r * ie b d
@@ -62749,6 +62819,8 @@
 tepidity t e . p * i d i t i
 tepidly t * e p i d l i
 tepidness t * e p i d n a s
+terabyte t * e r a b ie t
+terabytes t * e r a b ie t s
 tercentenaries t ~ er s e n . t * ee n a r i z
 tercentenary t ~ er s e n . t * ee n a r i
 tercentennial t ~ er s e n . t * e n eer l
@@ -63493,6 +63565,8 @@
 timeless t * ie m l a s
 timelier t * ie m l eer r
 timeliest t * ie m l i i s t
+timeline t * ie m l ie n
+timelines t * ie m l ie n z
 timeliness t * ie m l i n a s
 timely t * ie m l i
 timepiece t * ie m p ee s
@@ -64055,6 +64129,8 @@
 touchingly t * u ch i ng l i
 touchline t * u ch l ie n
 touchlines t * u ch l ie n z
+touchscreen t * u ch + s k r ee n
+touchscreens t * u ch + s k r ee n z
 touchstone t * u ch + s t oa n
 touchstones t * u ch + s t oa n z
 touchy t * u ch i
@@ -66296,6 +66372,10 @@
 uninspired ~ u n i n . s p * ie a d
 uninspiring ~ u n i n . s p * ie a r i ng
 uninsured ~ u n i n . sh * oor d
+uninstall ~ u n i n . s t * aw l
+uninstalled ~ u n i n . s t * aw l d
+uninstalling ~ u n i n . s t * aw l i ng
+uninstalls ~ u n i n . s t * aw l z
 unintelligent ~ u n i n . t * e l i j a n t
 unintelligently ~ u n i n . t * e l i j a n t l i
 unintelligible ~ u n i n . t * e l i j a b l
@@ -69229,6 +69309,7 @@
 wielding w * ee l d i ng
 wields w * ee l d z
 wife w * ie f
+wifi w ie f * ie
 wifelier w * ie f l eer r
 wifeliest w * ie f l i i s t
 wifelike w * ie f l ie k
