diff -u text710.edin.orig text710.edin
--- text710.edin.orig	2010-11-04 18:01:12.000000000 +0300
+++ text710.edin	2010-11-04 18:13:24.000000000 +0300
@@ -1009,7 +1009,6 @@
 Dingle d * i ng g l
 Dinnington d * i n i ng t a n
 Dip d * i p
-Dir d i . r * e k t a r
 Dives d * ie v ee z
 Djibouti j i . b * uu t i
 Djiboutian j i . b * uu t eer n
@@ -1434,6 +1433,7 @@
 Gov g * u v a n a r
 Governor-General g ~ u v n a - j * e n r a l
 Governor-Generals g ~ u v n a - j * e n r a l z
+Graham g r * aa m
 Grampian g r * aa m p eer n
 Granada g r a . n * ar d a
 Grangemouth g r * ai n j m a th
@@ -2596,7 +2596,6 @@
 Nicaragua n ~ i k a . r * aa g y oor
 Nicaraguan n ~ i k a . r * aa g y oor n
 Nicaraguans n ~ i k a . r * aa g y oor n z
-Nice n * ee s
 Nicholas n * i k a l a s
 Nick n * i k
 Nicola n * i k a l a
@@ -7037,7 +7036,6 @@
 attaboy ~ aa t a . b * oi
 attach a . t * aa ch
 attache a . t * aa sh ai
-attaches a . t * aa sh ai z
 attached a . t * aa ch t
 attaches a . t * aa ch i z
 attaching a . t * aa ch i ng
@@ -11221,6 +11219,8 @@
 byroads b * ie r oa d z
 bystander b * ie s t aa n d a r
 bystanders b * ie s t aa n d a z
+byte b * ie t
+bytes b * ie t s
 byway b * ie w ai
 byways b * ie w ai z
 byword b * ie w er d
@@ -24200,8 +24200,6 @@
 exports * e k s p aw t s
 exports i k . s p * aw t s
 expos * e k s p oa z
-expose e k . s p * oa z ai
-exposes e k . s p * oa z ai z
 expose i k . s p * oa z
 exposed i k . s p * oa z d
 exposes i k . s p * oa z i z
@@ -28111,6 +28109,8 @@
 gifting g * i f t i ng
 gifts g * i f t s
 gig g * i g
+gigabyte g * i g a b ie t
+gigabytes g * i g a b ie t s
 gigantic j ie . g * aa n t i k
 giggle g * i g l
 giggled g * i g l d
@@ -35214,6 +35214,8 @@
 kilns k * i l n z
 kilo k * ee l oa
 kilo- k * i l a -
+kilobyte k * i l a b ie t
+kilobytes k * i l a b ie t s
 kilocycle k * i l a s ie k l
 kilocycles k * i l a s ie k l z
 kilogram k * i l a g r aa m
@@ -37857,6 +37859,7 @@
 make-ups m * ai k - u p s
 maker m * ai k a r
 makers m * ai k a z
+makefile m ~ ai k . f * ie l
 makes m * ai k s
 makeshift m * ai k sh i f t
 makeshifts m * ai k sh i f t s
@@ -38780,6 +38783,8 @@
 meeting-places m * ee t i ng - p l ai s i z
 meetings m * ee t i ng z
 meets m * ee t s
+megabyte m * e g a b ie t
+megabytes m * e g a b ie t s
 megacycle m * e g a s ie k l
 megacycles m * e g a s ie k l z
 megadeath m * e g a d e th
@@ -42388,6 +42393,7 @@
 officiousness a . f * i sh a s n a s
 offing * o f i ng
 offish * o f i sh
+offline o f . l * ie n
 offprint * o f p r i n t
 offprints * o f p r i n t s
 offset * o f s e t
@@ -42538,6 +42544,7 @@
 ongoing * o n g oa i ng
 onion * u n eer n
 onions * u n eer n z
+online o n . l * ie n
 onlooker * o n l oo k a r
 onlookers * o n l oo k a z
 only * oa n l i
@@ -47087,6 +47094,7 @@
 port p * aw t
 portability p ~ aw t a . b * i l i t i
 portable p * aw t a b l
+portably p * aw t a b l i
 portage p * aw t i j
 portages p * aw t i j i z
 portal p * aw t l
@@ -49773,8 +49781,6 @@
 rechauffes r ai . sh * oa f ai z
 regime r ai . zh * ee m
 regimes r ai . zh * ee m z
-resume r * e z y uu m ai
-resumes r * e z y uu m ai z
 rabbi r * aa b ie
 rabbinical r a . b * i n i k l
 rabbis r * aa b ie z
@@ -50394,6 +50400,7 @@
 readjustment r ~ ee a . j * u s t m a n t
 readjustments r ~ ee a . j * u s t m a n t s
 readjusts r ~ ee a . j * u s t s
+readme r * ee d . m ee
 readmission r ~ ee a d . m * i sh n
 readmissions r ~ ee a d . m * i sh n z
 readmit r ~ ee a d . m * i t
@@ -50556,7 +50563,6 @@
 rebuttals r i . b * u t a l z
 rebutted r i . b * u t i d
 rebutting r i . b * u t i ng
-rec r i . s * ee v d
 recalcitrance r i . k * aa l s i t r a n s
 recalcitrancy r i . k * aa l s i t r a n s i
 recalcitrant r i . k * aa l s i t r a n t
@@ -50636,6 +50642,11 @@
 recessionals r i . s * e sh a n l z
 recessions r i . s * e sh n z
 recessive r i . s * e s i v
+recharge r i . ch * ar j
+rechargeable r i . ch * ar j a b l
+recharged r i . ch * ar j d
+recharges r i . ch * ar j i z
+recharging r i . ch * ar j i ng
 recherche r a . sh * air sh ai
 recidivism r i . s * i d i v i z a m
 recidivist r i . s * i d i v i s t
@@ -53571,6 +53582,7 @@
 runny r * u n i
 runs r * u n z
 runt r * u n t
+runtime r ~ u n . t * ie m
 runts r * u n t s
 runway r * u n w ai
 runways r * u n w ai z
@@ -60716,6 +60728,7 @@
 subpoenaed s a . p * ee n a d
 subpoenaing s a . p * ee n a r i ng
 subpoenas s a . p * ee n a z
+subroutine s ~ u b . r uu . t * ee n
 subs s * u b z
 subscribe s a b . s k r * ie b
 subscribed s a b . s k r * ie b d
@@ -62749,6 +62762,8 @@
 tepidity t e . p * i d i t i
 tepidly t * e p i d l i
 tepidness t * e p i d n a s
+terabyte t * e r a b ie t
+terabytes t * e r a b ie t s
 tercentenaries t ~ er s e n . t * ee n a r i z
 tercentenary t ~ er s e n . t * ee n a r i
 tercentennial t ~ er s e n . t * e n eer l
